packages/experimental-utils/src/ast-utils/eslint-utils/ReferenceTracker.ts

```diff
@@ -4,9 +4,12 @@ import * as eslintUtils from 'eslint-utils';
 import { TSESTree } from '../../ts-estree';
 import * as TSESLint from '../../ts-eslint';
 
-const ReferenceTrackerREAD: unique symbol = eslintUtils.ReferenceTracker.READ;
-const ReferenceTrackerCALL: unique symbol = eslintUtils.ReferenceTracker.CALL;
-const ReferenceTrackerCONSTRUCT: unique symbol =
+// TODO: FILE ISSUE(S) SOMEWHERE
```

Comment:
Looks like this is just... how it has to be! https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html